packages/fuel-indexer-database/postgres/src/lib.rs

```diff
@@ -162,30 +169,30 @@ pub async fn graph_root_latest(
     #[cfg(feature = "metrics")]
     METRICS.db.postgres.graph_root_latest_calls.inc();
 
-    let row = sqlx::query(&format!(
-        r#"
-        SELECT *
+    let row = sqlx::query(
+        "SELECT 
+            id, 
+            version, 
+            schema_name, 
+            query,
+            schema, 
+            schema_identifier
         FROM graph_registry_graph_root
-        WHERE schema_name = '{namespace}' AND schema_identifier = '{identifier}'
-        ORDER BY id DESC LIMIT 1"#
-    ))
+        WHERE schema_name = $1 AND schema_identifier = $2
+        ORDER BY id DESC LIMIT 1",
+    )
+    .bind(namespace)
+    .bind(identifier)
     .fetch_one(conn)
     .await?;
 
-    let id: i64 = row.get(0);
-    let version: String = row.get(1);
-    let schema_name: String = row.get(2);
-    let query: String = row.get(3);
-    let schema: String = row.get(4);
-    let schema_identifier: String = row.get(5);
-
     Ok(GraphRoot {
-        id,
-        version,
-        schema_name,
-        query,
-        schema,
-        schema_identifier,
+        id: row.get("id"),
```

Comment:
Meh, I did prefer the previous version here as it made the types explicit and looked cleaner. @deekerno thoughts?

Reply:
I agree. I'm surprised that it actually worked because `row.get(..)` has never worked any time that I've excluded the type annotation. 🤔

Reply:
I was trying to save memory by not initialising variables where we can just pass the value in directly, but the performance difference here is probably minimal. To my eyes the latter looks cleaner, as there are less lines to read and it expresses the same logic, but I don't mind at all, I'll switch to the former way!

Reply:
Yeah, I know we don't _always_ annotate types but I think it's good practice to do so, especially in these queries where we're pulling things out of the raw result.

Reply:
@rvmelkonian 

Memory-wise these 2 things are the same

```
# 1
let bar = Bar{ foo: row.get(0) };

# 2
let foo: String = row.get(0);
let bar = Bar{ foo };
```

In the second example `Bar` owns `foo` so no memory is copied. Number 2 is just more explicit

Reply:
you're absolutely right, It's my cursed golang brain kicking in!